src/Stratis.Bitcoin.Features.ColdStaking/ColdStakingManager.cs

```diff
@@ -225,14 +227,14 @@ internal HdAccount GetOrCreateColdStakingAccount(string walletName, bool isColdW
                 accountName = HotWalletAccountName;
             }
 
-            account = wallet.AddNewAccount(walletPassword, this.dateTimeProvider.GetTimeOffset(), accountIndex, accountName);
+            account = wallet.AddNewAccount(walletPassword,  accountIndex, accountName, this.dateTimeProvider.GetTimeOffset());
 
             // Maintain at least one unused address at all times. This will ensure that wallet recovery will also work.
-            IEnumerable<HdAddress> newAddresses = account.CreateAddresses(wallet.Network, 1, false);
-            this.UpdateKeysLookupLocked(newAddresses);
+            //IEnumerable<HdAddress> newAddresses = account.CreateAddresses(wallet.Network, 1, false);
```

Comment:
Please remove or add comment on why this commented out.

Reply:
Removed. The new wallet repository handles topping up of new addresses as soon as addresses become used and the related information is committed to the database. It is handled via the `TopUpTracker` class.